### DISCUSSION TITLE
Relative path when build package module on npm

### DISCUSSION THREAD

**Discussion by vitran12** [chakra-ui/panda#2432] [Q&A]
Hi everyone,

I'm building 1 package UI Library.

I use`tsc` && `tsc-alias` to build bundles.  I meet the error with the problem path. When I run by Next JS, it's okay but with Vite I meet when finish build: 

Before:

```jsx
import { splitCssProps } from '@/styled-system/jsx';
```
After:

```jsx
node_modules/test-build-ui/dist/esm/components/ui/switch/Switch.js:5:30:
      5 │ import { splitCssProps } from '../../../styled-system/jsx';
```
Hope everyone helps me.

**Comment by astahmer** [comment]
hey, this is unrelated to panda and is because you're using tsconfig path aliases which are not supposed to be the source of truth, you need a bundler plugin to fix the build output such as https://www.npmjs.com/package/vite-tsconfig-paths

prefer using the standard package.json when possible https://nodejs.org/api/packages.html#imports
